contrib/exporters/secadvisor/mod/transform.go

```diff
@@ -200,12 +139,22 @@ func (ft *securityAdvisorFlowTransformer) getNetwork(f *flow.Flow) *SecurityAdvi
 		return nil
 	}
 
+	aName, err := ft.resolver.IPToName(f.Network.A, f.NodeTID)
+	if err != nil {
+		aName = f.Network.A
```

Comment:
I worry that this will break existing Security Advisor processing that expects either `""` or `"0_0_podname_0"` in this field. I prefer to leave the value of `aName` as an empty string in this case. (same for `bName` below)

Note that in any case we're planning to deprecate `aName` and `bName` and add pod/container information in a separate `Context` section of the flow.

Reply:
HI @dubek: fyi, we have a field 'version' exactly for this make sure that we correlate we the correct version of SA.